CONTRIBUTING.md

```diff
@@ -11,19 +11,18 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
 
 ## Requesting An Icon
 
-We welcome icon requests. However, before you submit a new issue please make sure the icon:
+We welcome icon requests, before you submit a new issue please make sure the icon:
```

Comment:
These are two separate statements. Therefore, a period (or semicolon) would be more appropriate punctuation than a comma.

Reply:
Got it, will update this when there is more to change 👍